docs/reference/language/context.md

```diff
@@ -25,38 +25,180 @@ in some places that are also aware of their location in the document:
 [Show rules]($styling/#show-rules) provide context[^1] and numberings in the
 outline, for instance, also provide the proper context to resolve counters.
 
-## Style context
-With set rules, we can adjust style properties for parts or the whole of our
-document. We cannot access these without a known context, as they may change
-throughout the course of the document. When context is available, we can
-retrieve them simply by accessing them as fields on the respective element
-function.
+## Behavior of the context keyword
+Style properties frequently change within a document, for example by applying set 
+rules. To retrieve such poperties in a consistent way, one must first specify 
+the context where the query is to be executed. This is the purpose of the 
+`context` keyword. Once the context has been fixed, the property information 
+is available through a simple field access syntax. For example, `text.lang` 
```

Comment:
```suggestion
is available through field access syntax. For example, `text.lang` 
```

Reply:
You should be proud of how simple it is (in comparison to the competition :-).

Reply:
:D I'm trying to reduce the use of "simple" in my writing as things are differently simple/hard for different people and it's typically unnecessary fluff.